crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -24,9 +24,21 @@ pub struct StandardMaterial {
     #[texture(1)]
     #[sampler(2)]
     pub base_color_texture: Option<Handle<Image>>,
+
     // Use a color for user friendliness even though we technically don't use the alpha channel
     // Might be used in the future for exposure correction in HDR
+    /// Color the material "emits" to the camera.
+    ///
+    /// This is typically used for monitor screens or lamppost.
+    /// Anything that should be bright even if in a dark area.
+    ///
+    /// The brightness of the emissive color depends on how light it is.
+    /// The lighter, the more part the emissive color takes into the final result.
+    ///
+    /// Beware an emissive material won't light up surrounding areas,
+    /// you need to add a light source for that.
```

Comment:
Bloom is coming soon™️ ... :)

Reply:
Well, still technically doesn't light up the neighboring models, only the neighboring _pixels_ ;)